src/views/pack-create/pages/Step1SelectItemsPage/ui/Step1SelectItemsPage.tsx

```
@@ -0,0 +1,71 @@
+    () => selectedItems.map((item) => item.id),
+    [selectedItems]
+  );
+
+  const canGoNext = selectedItems.length > 0;
+
+  const openItemAdd = () => setIsAddingItems(true);
+  const closeItemAdd = () => setIsAddingItems(false);
+
+  const applySelectedItems = (items: typeof selectedItems) => {
+    resetSelectedItems();
+    items.forEach((item) => addSelectedItem(item));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)